protocol/contracts/leaser/src/migrate.rs

```diff
@@ -371,7 +377,7 @@ mod test {
         vec![Ok(cust1), Ok(cust2), Ok(cust3), Ok(cust4)].into_iter()
     }
 
-    fn migrate_msg() -> impl Fn() -> MigrateMsg {
-        || MigrateMsg {}
```

Comment:
Just as a note about the previous implementation, those are totally unnecessary gymnastics because passing the function pointer would've done that same without introducing indirection. I hope it will stay the way the new one is implemented.

Reply:
this is an old chunk of code and do net see any sense of commenting it now